cgi/members/memberentry.pl

```diff
@@ -605,7 +605,8 @@ BEGIN
 #            -default => $default,
 #        );
 my $CGIbranch = [];
-foreach(keys %$branches) {
+# Sort branches by branch name before generating CGI branch array
+foreach (sort {lc(%$branches{$a}{branchname}) cmp lc(%$branches{$b}{branchname})} keys %$branches) {
```

Comment:
This doesn't quite compile. The proper way to dereference `$branches` and its contents in this instance is either `$$branches{$a}{branchname}` or `$branches->{$a}{branchname}`.